inception-imls-opennlp/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/ner/OpenNlpNerRecommender.java

```diff
@@ -227,13 +227,21 @@ private NameSample createNameSample(CAS aCas, AnnotationFS aSentence,
         List<AnnotationFS> annotations = selectCovered(annotationType, aSentence);
         int numberOfAnnotations = annotations.size();
         List<Span> result = new ArrayList<>();
+        int maxSeen = 0;
```

Comment:
`maxSeen` sounds more like a count than an index. I would call it something like `largestEndSoFar` or so

Reply:
Variable has been renamed